scripts/zfs.sh

```diff
@@ -42,10 +43,11 @@ OPTIONS:
 	-h      Show this message
 	-v      Verbose
 	-u      Unload modules
+	-t      Enable kernel stack tracer
```

Comment:
nit: I suppose this can be "-S" also, for consistency

Reply:
I was going back and forth on this.  Will make the change.

Reply:
Latest commit with this change. Thanks!